JM/CF/Scripts/3_Game/CommunityFramework/Spline/CF_Spline.c

```diff
@@ -0,0 +1,264 @@
+class CF_Spline : Managed /*CF_ObservableCollection*/
+{
+	autoptr ScriptInvoker Update = new ScriptInvoker();
```

Comment:
Maybe `OnUpdate`? Is this maybe worth a `CF_EventHandler`?

What is this for? Not documented inline or in docs/

Reply:
It's for updating every point (node) in the spline. As one position change can affect the neighbors, and that affects its neighbors and so on, and so on.

Tangents all need to be recalculated

Reply:
Alright, still think it should be renamed to `OnUpdate`